backend/webhooks/events/events.go

```diff
@@ -5,14 +5,16 @@ import "github.com/teamhanko/hanko/backend/persistence/models"
 type Event string
 
 const (
-	User         Event = "user"
-	UserCreate   Event = "user.create"
-	UserUpdate   Event = "user.update"
-	UserDelete   Event = "user.delete"
-	Email        Event = "user.update.email"
-	EmailCreate  Event = "user.update.email.create"
-	EmailPrimary Event = "user.update.email.primary"
-	EmailDelete  Event = "user.update.email.delete"
+	User             Event = "user"
+	UserCreate       Event = "user.create"
+	UserUpdate       Event = "user.update"
+	UserDelete       Event = "user.delete"
+	UserEmail        Event = "user.update.email"
+	UserEmailCreate  Event = "user.update.email.create"
+	UserEmailPrimary Event = "user.update.email.primary"
+	UserEmailDelete  Event = "user.update.email.delete"
+
+	EmailCreate Event = "email.create"
```

Comment:
It is called `email.create` because it is a generic webhook event. In the future, when we send more emails (e.g. when the password changed, ...) you do not need to subscribe to multiple events, just one.